src/session/components/message_entry.rs

```diff
@@ -216,6 +228,40 @@ impl MessageEntry {
         self.imp().text_view.buffer().set_text(&text);
     }
 
+    /* formatted_text is being replaced every time text buffer has changed,
+     * so it's very unefficient to parse markdown text when it's not nesessary,
+     * instead we should use a separate method, that will be called only when composing a message */
```

Comment:
What does this mean? Isn't this parsing function just called when we send a message or save a draft message? I think this comment can be removed.

Reply:
That was an argumentation why to create `MessageEntry::parsed_formatted_markdown()`, but you're right it was useless